src/apps/service/apps/job_listings.clj

```diff
@@ -58,7 +64,7 @@
     :parent_id       parent-id
     :batch           (:is-batch job)
     :batch_status    (when (:is-batch job) (format-batch-status id))
-    :can_share       (and (nil? parent-id) (job-permissions/job-steps-support-job-sharing? apps-client (rep-steps id)))}))
+    :can_share       (job-supports-sharing? apps-client rep-steps job)}))
```

Comment:
This doesn't seem related to system IDs.
Is this a bug-fix? Or just a minor refactor?